### DISCUSSION TITLE
Sound through ear speaker instead of main speakers

### DISCUSSION THREAD

**Discussion by FicoAG** [bluefireteam/audioplayers#1316] [Q&A]
Hi, anybody else having this issue? Not sure since when (it wasn't the case before), on iPhone the audio is playing throng the ear speaker instead of the main speakers. On iPads its working fine. Any idea?

**Comment by alialamrad** [comment]
any soluation ? I have the same problem !

**Comment by Gustl22** [comment]
See #1194

**Reply by alialamrad** [reply]
many thanks
